### ISSUE TITLE
KeyCloak authorization service : officialy released or not ?

### ISSUE THREAD

**Issue by semangard** [mrparkers/terraform-provider-keycloak#416]
Hello,

There isn't any doc on https://registry.terraform.io/providers/mrparkers/keycloak/latest/docs
about KeyCloak Authorization Services
despite resources seems to be available with some few examples 

==> Is that officially released or not ? 
What is the coverage of these resources compared to all KC features ?
When do you plan to publish the doc ?

Regards

**Comment by mrparkers** [comment]
You're right, a lot of the authz features that are implemented within this provider are missing documentation.  You can look here for a basic example of what exists and how to use it: https://github.com/mrparkers/terraform-provider-keycloak/blob/master/example/client_authorization_policys.tf

I've tried to backfill missing docs when I've had time, but I've never actually used the authz features of Keycloak, so it's a bit of a struggle for me to document these features when I don't personally understand them myself 😅
